http/src/main/resources/init-resources/startup.sh

```diff
@@ -206,13 +206,23 @@ else
         # what was previously under $HOME will now appear in new $HOME as well
         docker exec $JUPYTER_SERVER_NAME /bin/bash -c "[ -d $JUPYTER_USER_HOME/notebooks ] && [ ! -d $JUPYTER_USER_HOME/notebooks/.jupyter ] && rsync -avr --progress --exclude notebooks . $JUPYTER_USER_HOME/notebooks || true"
 
-        # Make sure when runtimes restarts, they'll get a new version of jupyter docker compose file
-        $GSUTIL_CMD cp gs://${INIT_BUCKET_NAME}/`basename ${JUPYTER_DOCKER_COMPOSE}` $JUPYTER_DOCKER_COMPOSE
+        NEED_MIGRATE=$(docker exec $JUPYTER_SERVER_NAME /bin/bash -c "[ -d $JUPYTER_USER_HOME/notebooks ] && echo 'true' || echo 'false'")
```

Comment:
Maybe add a  comment - why does the presence of this directory indicate that it needs migration?

Reply:
think [this](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/init-resources/startup.sh#L128) is good enough?

Reply:
Oops, yep -missed that was above